{"author":{"id":"ca90045d5399c21b6727f972ab5a6a5357d5d0dfde83d0b744683b6068f30ba1"},"ops":[{"type":6,"timestamp":1607429341,"nonce":"MN6EuQJ6U3NV8ZefGf7afbqiM/Q=","metadata":{"github-id":"MDE1OlVzZXJDb250ZW50RWRpdElzc3VlRWRpdDo0NjU2NzQ2NDg="},"target":"ce97d78a65434298b15a7eb2f660d4cc529d82fbd0f15e85875a76d3b8f12710","message":"Hi,\n\nI'm facing the below issue while compiling it in Pi3 lite\n\n```\npi@raspberrypi:~/wifibroadcast $ make\ng++  -Wall -O2 -DWFB_VERSION='\"20.12.8.43634-8d2d97de\"' -std=gnu++11 -c -o src/rx.o src/rx.cpp\ncc  -Wall -O2 -DWFB_VERSION='\"20.12.8.43634-8d2d97de\"' -std=gnu99 -c -o src/radiotap.o src/radiotap.c\ncc  -Wall -O2 -DWFB_VERSION='\"20.12.8.43634-8d2d97de\"' -std=gnu99 -c -o src/fec.o src/fec.c\ng++  -Wall -O2 -DWFB_VERSION='\"20.12.8.43634-8d2d97de\"' -std=gnu++11 -c -o src/wifibroadcast.o src/wifibroadcast.cpp\ng++ -o wfb_rx src/rx.o src/radiotap.o src/fec.o src/wifibroadcast.o  -lrt -lpcap -lsodium\ng++  -Wall -O2 -DWFB_VERSION='\"20.12.8.43634-8d2d97de\"' -std=gnu++11 -c -o src/tx.o src/tx.cpp\ng++ -o wfb_tx src/tx.o src/fec.o src/wifibroadcast.o  -lrt -lpcap -lsodium\ncc  -Wall -O2 -DWFB_VERSION='\"20.12.8.43634-8d2d97de\"' -std=gnu99 -c -o src/keygen.o src/keygen.c\ncc -o wfb_keygen src/keygen.o  -lrt -lpcap -lsodium\nPYTHONPATH=`pwd` trial telemetry.tests\ntelemetry.tests.test_proxy\n  UDPProxyTestCase\n    test_proxy ...                                                         [OK]\n    test_rssi_injection ...                                                [OK]\ntelemetry\n  tests\n    test_tuntap ...                                                     [ERROR]\n\n===============================================================================\n[ERROR]\nTraceback (most recent call last):\n  File \"/usr/lib/python2.7/dist-packages/twisted/trial/runner.py\", line 531, in loadPackage\n    module = modinfo.load()\n  File \"/usr/lib/python2.7/dist-packages/twisted/python/modules.py\", line 392, in load\n    return self.pathEntry.pythonPath.moduleLoader(self.name)\n  File \"/usr/lib/python2.7/dist-packages/twisted/python/reflect.py\", line 308, in namedAny\n    topLevelPackage = _importAndCheckStack(trialname)\n  File \"/usr/lib/python2.7/dist-packages/twisted/python/reflect.py\", line 255, in _importAndCheckStack\n    reraise(excValue, excTraceback)\n  File \"/home/pi/wifibroadcast/telemetry/tests/test_tuntap.py\", line 17, in \u003cmodule\u003e\n    from telemetry.tuntap import TUNTAPTransport, TUNTAPProtocol\n  File \"/home/pi/wifibroadcast/telemetry/tuntap.py\", line 41, in \u003cmodule\u003e\n    from pyroute2 import IPRoute\nexceptions.ImportError: No module named pyroute2\n\ntelemetry.tests.test_tuntap\n-------------------------------------------------------------------------------\nRan 3 tests in 1.153s\n\nFAILED (errors=1, successes=2)\nmake: *** [Makefile:43: test] Error 1\n```","files":null},{"type":3,"timestamp":1607429670,"nonce":"BIhdYX2JCkBKhOnnKTDenaA5Qg0=","metadata":{"github-id":"MDEyOklzc3VlQ29tbWVudDc0MDU4NDE0MQ==","github-url":"https://github.com/svpcom/wfb-ng/issues/68#issuecomment-740584141"},"message":"sorry, i just see the dependency list in stdeb.cfg\ni would suggest to put it in README.md","files":null},{"type":4,"timestamp":1607429670,"nonce":"OPxBT6lyYiFRjA6cwZXXFtYMJLE=","metadata":{"github-id":"MDExOkNsb3NlZEV2ZW50NDA4NDQ4NDQyOQ=="},"status":2}]}